tests/test_memory_core/test_memory_core.py

```diff
@@ -450,119 +420,8 @@ def db_auto(stride_0, stride_1, stride_2, stride_3, stride_4, stride_5,
                                flags=["-Wno-fatal"])
 
 
-def db_basic(stride_0, stride_1, stride_2, stride_3, stride_4, stride_5,
```

Comment:
Why is this deleted?

Reply:
I had a secondary mode that was unused (basically ignored read/write signals, but it wouldn't be useful in this case)